ONSdigital/ssdc-rm-support-tool
# Limit print suppliers

Why?
The print supplier field in the support tool was previously a free text box, this is not a good idea.

What?
The support tool backend now has a config item in application.yml specifying print suppliers and a new endpoint to serve these up.
The support tool frontend now gives the choice of print supplier to use, retrieved from the aforementioned print supplier endpoint.

How to test?
Run the frontend and backend on this branch:
create a survey
create a collection exercise
create wave of contact - the print supplier should now display as a dropdown giving the options that are in the print supplier config in the backend.

Links?
https://trello.com/c/6tMnh7n3/2388-limit-print-suppliers-in-ui-based-on-print-file-service-config-8
